samcli/local/lambda_service/service.py

```diff
@@ -0,0 +1,128 @@
+"""Local Lambda Service"""
+
```

Comment:
Can we rename the file to something else? The word "service" is very generic, even though I know the file is creating a service

Reply:
We can. :)